WooCommerce/Classes/POS/Models/PointOfSaleAggregateModel.swift

```diff
@@ -196,6 +196,16 @@ extension PointOfSaleAggregateModel {
         }
     }
 
+    @MainActor
+    func sendReceipt(to emailAddress: String) async {
+        guard let order = orderController.order else {
+            return
+        }
+        // TODO:
+        // Add eligiblity for correct WC and WCPay versions
```

Comment:
This will need to be added before the feature flag is enabled; at the moment, on unsupported versions you tap the button and nothing happens.